plugins/main/bridge/bridge.go

```diff
@@ -46,17 +46,18 @@ const defaultBrName = "cni0"
 
 type NetConf struct {
 	types.NetConf
-	BrName       string `json:"bridge"`
-	IsGW         bool   `json:"isGateway"`
-	IsDefaultGW  bool   `json:"isDefaultGateway"`
-	ForceAddress bool   `json:"forceAddress"`
-	IPMasq       bool   `json:"ipMasq"`
-	MTU          int    `json:"mtu"`
-	HairpinMode  bool   `json:"hairpinMode"`
-	PromiscMode  bool   `json:"promiscMode"`
-	Vlan         int    `json:"vlan"`
-	MacSpoofChk  bool   `json:"macspoofchk,omitempty"`
-	EnableDad    bool   `json:"enabledad,omitempty"`
+	BrName             string `json:"bridge"`
+	IsGW               bool   `json:"isGateway"`
+	IsDefaultGW        bool   `json:"isDefaultGateway"`
+	ForceAddress       bool   `json:"forceAddress"`
+	IPMasq             bool   `json:"ipMasq"`
+	MTU                int    `json:"mtu"`
+	HairpinMode        bool   `json:"hairpinMode"`
+	PromiscMode        bool   `json:"promiscMode"`
+	Vlan               int    `json:"vlan"`
+	DisableDefaultVlan bool   `json:"disableDefaultVlan"`
```

Comment:
could we have a `preserveDefaultVlan` attribute instead ? 

That would allow a simpler configuration when people want to override the default VLAN (... I'm taking an educated guess, but I would say that's the most common use-case) , and I also think (generically speaking) a boolean argument should have "positive" logic - it should be about "enabling" something - enabling a flag to disable a feature is more confusing than the alternative.

@mlguerrero12

Reply:
yes, I agree. My first choice was enableDefaultVlan, but I was having issues setting the default value equal to true (since the idea is to preserve the current behavior). Today, I tried again and it worked (another reason to not work late on Fridays).

So, you want me to change the name to preserveDefaultVlan, right?

Reply:
Yes. Maybe someone has a better name for it, but I think that represents the user's intentions. 

Thanks for listening / taking into account this suggestion.

Reply:
done